Issue using KTX-Software when fmt is already included

So I updated KTX-Software to a later release and it seems fmt as been added as a dependency to KTX-Software.
My issue is that I use KTX-Software as a submodule myself alongside fmt which makes my project break with:

[INFO] Adding FMT from directory: external/fmt
CMake Error at external/fmt/CMakeLists.txt:50 (add_library):
  add_library cannot create target "fmt" because another target with the same
  name already exists.  The existing target is a static library created in
  source directory
  "----/KTX-Software/other_projects/fmt".
  See documentation for policy CMP0002 for more details.
Call Stack (most recent call first):
  external/fmt/CMakeLists.txt:285 (add_module_library)

Is there a way you can make fmt for KTX-Software optional? Or supply with an already built target?

Best Regards

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1041,9 +1041,13 @@ else()
 endif()
 
 # Other external projects
-set(FMT_SYSTEM_HEADERS ON)
-add_subdirectory(other_projects/fmt)
-add_subdirectory(other_projects/cxxopts)
+if(KTX_FEATURE_TOOLS AND NOT TARGET fmt::fmt)
+    set(FMT_SYSTEM_HEADERS ON)
+    add_subdirectory(other_projects/fmt)
+endif()
+if(KTX_FEATURE_TOOLS AND NOT TARGET cxxopts::cxxopts)
+    add_subdirectory(other_projects/cxxopts)
+endif()
 
 # Tools
 if(KTX_FEATURE_TOOLS)

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1041,7 +1041,10 @@ else()
 endif()
 
 # Other external projects
-if(KTX_FEATURE_TOOLS AND NOT TARGET fmt::fmt)
+# `NOT TARGET`s here are a precaution. They would only be exercised if
+# this CMakeLists is included in another project which is unlikely
+# except for building the ktx library.
+if((KTX_FEATURE_TOOLS OR KTX_FEATURE_TESTS) AND NOT TARGET fmt::fmt)
     set(FMT_SYSTEM_HEADERS ON)
     add_subdirectory(other_projects/fmt)
 endif()

